agent/exploits/cve_2024_38077.py

```diff
@@ -147,7 +147,7 @@ def accept(self, target: definitions.Target) -> bool:
         Returns:
             bool: True if the target is likely vulnerable, False otherwise.
         """
-        return _connect_to_license_server(target.origin) is not None
+        return _connect_to_license_server(target.host) is not None
```

Comment:
Why the change?

Reply:
the `_connect_to_license_server` only needs the ip not the whole schema.